lib/ReactViews/Tools/PedestrianMode/MovementsController.tsx

```diff
@@ -0,0 +1,342 @@
+import { runInAction } from "mobx";
```

Comment:
I wonder why this is `tsx` and in `ReactViews` it is more for a `Map`? Personally think that this could be used a general feature in TerriaJS.